### PR TITLE
Update sections 1-6

### PR DESCRIPTION
**Pull request by gcharest**
Updated all sections with new content:

- Fix typos and syntax in all documents
- OSS Use: 
  - Move Copyright to OSS Contribution/Intellectual Property
- OSS Contribution: 
  - Define Contribution
  - Add Intellectual Property section with Copyright section from OSS Use
- Open Culture:
  - Add "Entrenched Behavior and Fear of the Unknown" section

### CODE REVIEW

**Comment by gcharest** [comment]
Known Link issue:
- 2018-07-15 22:09: http://www.unicode.org/versions/Unicode6.2.0/ is not responding.

**Comment by gcharest** [comment]
Updated all sections, FR/EN

File: en/3_Open_Source_Software_Use.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[« Previous Page : Open Standards](2_Open_Standards.md) | [Table of Contents](../README.md#table-of-contents) | [Next Page: Open Source Software Contribution »](4_Open_Source_Software_Contribution.md)
+[« Previous Page : Open Standards](2_Open_Standards.md) | [Table of Contents](../README.md#table-of-contents) | [Next Page: Open Source Software Contribution »](4_Open_Source_Software_Contribution)
```
Comment (nschonni) [inline en/3_Open_Source_Software_Use.md]: Droping the `.md` makes this invalid. Not sure why the build is passing with this

**Comment by gcharest** [comment]
Well, the good thing is that we may have found a bug with the test? Will
update all links.

Le lun. 16 juil. 2018 à 19:20, Nick Schonning <notifications@github.com> a
écrit :

**Comment by gcharest** [comment]
Should have paid more attention to the auto-complete feature :D

File: en/3_Open_Source_Software_Use.md (hunk 1 above)
Comment (gcharest) [reply en/3_Open_Source_Software_Use.md]: Thanks @nschonni ! Found a whole lot of other dead links! Really weird that the script didn't pick those up...

**Pull request merged**